scuba/config.py

```diff
@@ -73,20 +73,22 @@ def from_yaml(self, node):
 
 
 def find_config():
-    '''Search up the diretcory hierarchy for .scuba.yml
+    '''Search up the directory hierarchy for .scuba.yml
 
-    Returns: path, rel on success, or None if not found
+    Returns: path, rel, scuba_config on success, or None if not found
```

Comment:
This doesn't match the line added below.
```suggestion
    Returns: path, rel, config on success, or None if not found
```

Reply:
Good catch

Reply:
Addressed by TimothySprague/scuba@ea1be7f6fb2a25a586329f831373ff28e7a0a2db